### ISSUE TITLE
The prediction is not working properly

### ISSUE THREAD

**Issue by cahya-wirawan** [rajveermalviya/language-modeling#2]
**Describe the bug**
After the training is done, the prediction of next word (using python test.py) doesn't work at all, the prediction results are always the same for any input sentence

**To Reproduce**
1. python test.py

**Expected behavior**
The result of prediction should make sense

**Screenshots**
$ python test.py
...
Enter atleast 3 words: 
i am hungry
Word  hungry  does not exist

indexes of last 3 words : [68, 1632, 1]

prediction indexes      : [2, 0, 0, 0, 0, 0, 0, 0, 0, 0]

1 - i am hungry .
2 - i am hungry the
3 - i am hungry the
4 - i am hungry the
5 - i am hungry the
6 - i am hungry the
7 - i am hungry the
8 - i am hungry the
9 - i am hungry the
10 - i am hungry the

Enter atleast 3 words: 
medicine has existed for thousands of

indexes of last 3 words : [11, 1563, 4]

prediction indexes      : [2, 0, 0, 0, 0, 0, 0, 0, 0, 0]

1 - medicine has existed for thousands of .
2 - medicine has existed for thousands of the
3 - medicine has existed for thousands of the
4 - medicine has existed for thousands of the
5 - medicine has existed for thousands of the
6 - medicine has existed for thousands of the
7 - medicine has existed for thousands of the
8 - medicine has existed for thousands of the
9 - medicine has existed for thousands of the
10 - medicine has existed for thousands of the

Enter atleast 3 words: 
the world is not enough

indexes of last 3 words : [13, 63, 543]

prediction indexes      : [2, 0, 0, 0, 0, 0, 0, 0, 0, 0]

1 - the world is not enough .
2 - the world is not enough the
3 - the world is not enough the
4 - the world is not enough the
5 - the world is not enough the
6 - the world is not enough the
7 - the world is not enough the
8 - the world is not enough the
9 - the world is not enough the
10 - the world is not enough the

**Comment by cahya-wirawan** [comment]
More info:
I use Keras 2.1.6 and tensorflow-gpu 1.10.0. I changed also the batch_size in config file from 20 to 100 to speed up the training time.
The val_categorical_accuracy during the training is also very low (around 0.0481 from epoch 1 to 14)
Following is the result after epoch 14:
3099/3098 [==============================] - 348s 112ms/step - loss: 15.3376 - categorical_accuracy: 0.0484 - val_loss: 15.3423 - val_categorical_accuracy : 0.0481

**Comment by rajveermalviya** [comment]
@cahya-wirawan 
14 epochs are not good for the model, it was just done for testing purpose.
So, I have now also changed the [CONFIG.py](https://github.com/rajveermalviya/language-modeling/blob/master/CONFIG.py).
Also updated the [test.py](https://github.com/rajveermalviya/language-modeling/blob/master/test.py), the same word issue is fixed now.
Please let me know is there are any further issues. 
Thanks for filing an issue.
